tests/h/views/admin/users_test.py

```diff
@@ -240,8 +240,8 @@ def user_service(pyramid_config, db_session):
 
 
 @pytest.fixture
-def annotation_stats_service(pyramid_config, db_session):
-    service = Mock(spec_set=AnnotationStatsService(session=db_session))
+def annotation_stats_service(pyramid_config, pyramid_request):
+    service = Mock(spec_set=AnnotationStatsService(request=pyramid_request))
```

Comment:
I know you're just replacing an existing `Mock` with a `Mock` but changing this to something like `mock.create_autospec(AnnotationStatsService, instance=True, spec_set=True, request=pyramid_request)` would be better. You'd get a better-specced mock and also these tests would not be instantiating `AnnotationStatsService`